index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="Main">
+    <div class="One">
+      <img src="/photos/portflio.png" alt="Snapshoot">
+    </div>
+    <div class="Two">
+      <h1>Tonic</h1>
+    </div>
+    <div class="Three">
+      <ul class="skills">
+        <li class="can">CANOPY</li>
+        <li class="back">Back End Dev</li>
+        <li class="year">2005</li>
+      </ul>
+    </div>
+    <div class="Four">
+      <p> daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="Five">
+      <ul class="languages">
+        <li class="html">html</li>
+        <li class="css">css</li>
+        <li class="java">javaScript</li>
+      </ul>
+    </div>
+    <div class="Six">
+      <button class="butt">See Project</button>
+    </div>
+  </div>
```

Comment:
Your `works` section does not match [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design:
![image](https://user-images.githubusercontent.com/38088818/164945409-efd5b02f-0b3d-4a75-b8cf-7a9eab52b1b6.png)

Figma design:
![figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png?raw=true)

You must also include all four projects illustrated in [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).